robot/EDA/tests/browser/health_check/eda_health_check.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Documentation   Validate health check for EDA
+Resource        robot/EDA/resources/EDA.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/EDA/resources/SettingsHealthCheckPageObject.py
+
+Suite Setup     Open Test Browser
+Suite Teardown  Capture screenshot and delete records and close browser
+
+*** Test Cases ***
+Verify settings health check app is displayed
+    [Documentation]         Validates 'Settings Health Check" app is displayed under app tile
+    [tags]                  unstable        rbt:high        W-8864633
+    Open app launcher
+    Verify item exists       Settings Health Check
+
+Verify last run date is updated upon clicking run health check Button
+    [Documentation]         Validates the last run date is updated accordingly after clicking the
+    ...                     run health check button.
+    [tags]                  unstable        rbt:high        W-8874163
+    Reload Page
+    Select App Launcher Tab      Settings Health Check
```

Comment:
Is there a specific reason you want to navigate using '`Select app launcher`' ? If not, using a url to navigate will be stable and also quicker.

Reply:
Yes. We wanted to check the availability of this app under app launcher as a tab as its a requirement.Also to make sure to follow uniformity in 1st and 2nd test I am using this keyword. But I agree its little inconsistent. Let me monitor it few times and if I start seeing failure because of this keyword then I will change it